MonkeyWrench.Web.WebService/Scheduler/Scheduler.cs

```diff
@@ -317,13 +317,14 @@ private static void AddWork (DB db, List<DBHost> hosts, List<DBLane> lanes, List
 		{
 			DateTime start = DateTime.Now;
 			List<DBCommand> commands = null;
-			List<DBLaneDependency> dependencies = null;
+			List<DBLaneDependency> dependencies = DBLaneDependency_Extensions.GetDependencies (db, null);

```

Comment:
You can guarantee it's super lazy if you do:
```
var dependencies = new Lazy<List<DBLaneDependency>> (() => DBLaneDependency_Extensions.GetDependencies (db, null));
```
Then the first user of `dependencies.Value` would result in the query being run, and subsequent users would get the cached value. Probably no need to make that alteration now, but it can be a nicer pattern than bool-checking if the code is complicated.